resources/howtos/http2_ssl.md

```diff
@@ -0,0 +1,144 @@
+---
+permalink: /http2-ssl
+---
+
+# How to use HTTP2 and SSL with MeiliSearch
+
+For those who would like to use HTTP2 we have to warn you that this is **only possible if you have launched your server with SSL**.
+
+We will therefore see here how to launch a MeiliSearch server with SSL. In this little tutorial I will do it locally, but be aware that you can very well do the same thing on a server.
+
+To start I need the binary of MeiliSearch, we could also use docker, it would be necessary to pass it the parameters in environment variables aison that the certificates via a volume.
```

Comment:
```suggestion
To start I need the binary of MeiliSearch, we could also use docker, it would be necessary to pass it the parameters in environment variables and the certificates via a volume.
```